### PR TITLE
Add extruded mesh citation

### PR DESCRIPTION
**Pull request by doru1004**
Point to the arXiv article here: https://arxiv.org/abs/1604.05937

### CODE REVIEW

**Comment by wence-** [comment]
You should wire up the automated citation registration.

File: firedrake/citations.py [hunk 1]
```diff
@@ -146,3 +146,23 @@ def print_at_exit(cls):
   url =          {http://arxiv.org/abs/1604.05872}
 }
 """)
+
+Citations().add("Luporini2016", """
```
Comment (miklos1) [inline firedrake/citations.py]: Ah, another Luporini *et al.* paper.

File: docs/source/publications.rst [hunk 2]
```diff
@@ -24,9 +24,8 @@ If you cannot use this approach, there are a number of papers.  Those
 which are relevant depend a little on which functionality you used.
 
 For Firedrake itself, please cite :cite:`Rathgeber2015`.  If you use
-the :doc:`extruded mesh </extruded-meshes>` functionality, or
-quadrilateral meshes, please cite :cite:`McRae2014`.  Additionally,
-when using quadrilateral meshes, please also cite :cite:`Homolya2016`.
+the :doc:`extruded mesh </extruded-meshes>` functionality please cite :cite:`McRae2014` and :cite:`Bercea2016`.
```
Comment (miklos1) [inline docs/source/publications.rst]: Line breaks!

File: firedrake/citations.py (hunk 1 above)
Comment (wence-) [inline firedrake/citations.py]: IOW, this is wrong.

File: docs/source/_static/bibliography.bib [hunk 3]
```diff
@@ -171,3 +171,32 @@ @Article{Rathgeber2015
   eprint =       {1501.01809},
   url =          {http://arxiv.org/abs/1501.01809}
 }
+
+@Article{Luporini2016,
```
Comment (miklos1) [inline docs/source/_static/bibliography.bib]: Why are you adding this for a second time?

**Comment by miklos1** [comment]
Removed the duplicate bibliography entry, I think this now should be good to go.

**Comment by wence-** [comment]
OK.

**Pull request merged**